### PR TITLE
hack fixes

### PR DESCRIPTION
**Pull request by lightsofapollo**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -52,7 +51,6 @@ function ProfileBuilder(options) {
   }
 }
 
-
```
Comment (lambdabaa) [inline index.js:55]: Can you get rid of the whitespace changes :) ?

Comment (lambdabaa) [inline index.js:55]: Aside from that this looks good to me.

Comment (lambdabaa) [inline index.js:55]: @lightsofapollo Actually if you could name the commit something more like "Don't pass env vars to gaia make and fix baseProfile" that'd be cool too

**Comment by lightsofapollo** [comment]
I will send you a proper PR later with the changes I wanted to make... I want to remove the duplication here from marionette-profile-builder

**Pull request closed without merging**